tests/models/test_image.py

```diff
@@ -486,7 +486,7 @@ def test_image_badness(demo_image):
         assert demo_image.badness == 'Banding, Shaking, Bright Sky'
 
 
-# @pytest.mark.skip( reason="slow" )
+@pytest.mark.skip( reason="slow" )
```

Comment:
I want to keep it, and the corresponding one in test_source_list.py commented it... are you OK with that?

(I also have a slow one in test_psf.py, where it renders a whole bunch of psfs and make sure they look right.  That one's worth running every so often, but I don't know if it's worth running every time we change anything.)

Reply:
Usually when we start skipping things, we forget about it and never look back (there are many such tests in skyportal, for example). 

How about we add another flag (similar to INTERACTIVE), like SKIP_SLOW or something, and then you can decide when running it if you want to skip or not. 

Then in GA it will always run and that would be our failsafe.